src/main/java/racing/domain/CarMovableStrategy.java

```diff
@@ -6,14 +6,17 @@ public class CarMovableStrategy implements MovableStrategy {
 
     private final static int RANDOM_RANGE = 10;
     private final static int RACING_CONDITION = 4;
-    private final static Random RANDOM = new Random();
+    private final static int RUN = 1;
```

Comment:
```suggestion
    private static final int RUN = 1;
```
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers) 확인 부탁드릴게요!

Reply:
```
public protected private abstract default static final transient volatile synchronized native strictfp
```

순서대로 모두가 이해할만한 순서로 수정하도록 하겠습니다